### PR TITLE
feat(cli): expose existing_token client property

### PR DESCRIPTION
**Pull request by tumido**
**Description of your changes:**

This PR exposes `existing_token` option of `kfp.Client` to the CLI.

Resolves: https://github.com/kubeflow/pipelines/issues/11399

**Checklist:**
- [x] You have [signed off your commits](https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits)
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by hbelmiro — COMMENTED** [review]
/lgtm
/ok-to-test

**Comment by tumido** [comment]
@hbelmiro can you please approve the CI to be run on this? It seems `ok-to-test` is not enough and some additional button clicking or keyboard smashing is still required. 🤷‍♂️ 🙂

**Review by DharmitD — COMMENTED** [review]
@HumairAK could you approve the CI runs for this PR? Otherwise,
/lgtm

File: sdk/python/kfp/cli/cli.py [hunk 1]
```diff
@@ -85,6 +85,9 @@ def _install_completion(shell: str) -> None:
 @click.option(
     '--other-client-secret',
     help=parsing.get_param_descr(client.Client, 'other_client_secret'))
+@click.option(
+    '--existing-token',
+    help=parsing.get_param_descr(client.Client, 'existing_token'))
```
Comment (HumairAK) [inline sdk/python/kfp/cli/cli.py:90]: Thanks @tumido can you also add this change to: https://github.com/kubeflow/pipelines/blob/master/sdk/RELEASE.md#features ?

**Review by hbelmiro — COMMENTED** [review]
/lgtm

**Comment by HumairAK** [comment]
/lgtm
/approve

@tumido thank you! if you are up for it there are some other low hanging fruits here as well that can help with connecting against endpoints, I'm mainly thinking of `ssl_ca_cert` and `verify_ssl` if you're up for it 🥲

**Pull request merged**